gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
This shouldn't be here -- gradle should be installed on a per-project basis. (I've got lots of projects using gradle, and they're not all at the same version.)